packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -0,0 +1,235 @@
+            },
+            "&:focus": {
+                outline: "none"
+            }
+        }
+    };
+}
+
+export function applyPopupHeadingStyles(): ICSSRules<{}> {
+    return {
+        position: "relative",
+        "& > div": {
+            display: "flex",
+            borderBottom: `${toPx(1)} solid ${colors.border}`,
+            "& h3": {
+                flexGrow: "1",
+                "& + button": {
+                    fontSize: toPx(14),
+                    background: "none",
+                    border: "none",
+                    padding: `${toPx(2)} ${toPx(12)}`,
+                    color: "#0078D7",
+                    "&:focus": {
+                        outline: "none"
+                    },
+                    "&:hover": {
+                        textDecoration: "underline"
+                    },
+                    "&[aria-expanded='true']": {
+                        background: colors.menuGray,
+                        borderLeftColor: colors.lightBorder,
+                        borderRadius: `${toPx(2)} ${toPx(2)} 0 0`,
+                        borderRightColor: colors.lightBorder,
+                        borderTopColor: colors.lightBorder,
+                        color: colors.black,
+                        "&:hover": {
+                            textDecoration: "none"
+                        }
+                    }
+                }
+            }
+        }
+    };
+}
+
+export function applyInputBackplateStyle(): ICSSRules<{}> {
+    return {
+        appearance: "none",
+        height: toPx(36),
+        width: toPx(36),
+        backgroundColor: "transparent",
+        "&:focus": {
+            outline: "none",
+        }
+    };
+}
+
+export function applySelectInputStyles(): ICSSRules<{}> {
+    return {
+        flexGrow: "1",
+        lineHeight: toPx(16),
+        fontSize: toPx(14),
+        backgroundColor: colors.grayBackground,
+        borderRadius: toPx(2),
+        boxShadow: `inset ${toPx(0)} ${toPx(0)} ${toPx(4)} ${colors.boxShadow}`,
+        appearance: "none",
+        padding: localizeSpacing(Direction.ltr)(`${toPx(8)} ${toPx(24)} ${toPx(8)} ${toPx(8)}`),
+        border: "none",
+        outline: "none",
+        "&:-ms-expand": {
+            display: "none"
+        }
+    };
+}
+
+export function applySelectSpanStyles(): ICSSRules<{}> {
+    return {
+        position: "relative",
+        display: "flex",
+        flexGrow: "1",
+        "&:before, &:after": {
+            content: "''",
+            position: "absolute",
+            top: toPx(12),
+            zIndex: 1,
```

Comment:
is this actually getting rendered as a CSS value? I was under the impression JSS always wanted everything surrounded by quotes

Reply:
Fixed.